docs/backend/guides/how-to/create-a-custom-linter.md

```diff
@@ -0,0 +1,137 @@
+# How to Create a Custom Go Linter
+
+Creating custom GO linters can be a great way to analyze your 
+project's source code and alert you to bugs, errors, 
+or other issues with your code.
+
+## Setting up your linter
+
+### File structure
+Start by setting up the files you'll need for your linter:
+```golang
+-cmd
+    -linter_name
+        -main.go
+
+-pkg
+    -linter_name
+        -linter.go
+        -linter_test.go
+```
+In the `cmd` folder create a folder for your linter called `<linter_name>` and add an empty file called `main.go`
+
+In `pkg` folder create another folder for your linter called `<linter_name>` and in it you will 
+have your `linter.go` and `linter_test.go`
```

Comment:
Should this specify that `linter.go` and `linter_test.go` are just examples (similar to how the folder names are).